src/templates/debug.html

```diff
@@ -27,8 +27,8 @@
     <select name="region" id="debug-region">
       {% set user_region = user.get_setting('region_override') %}
       <option value="" {{ 'selected' if not user_region }}>---</option>
-      {% for code, region in regions %}
-        <option value="{{ code }}" {{ 'selected' if code == user_region }}>
+      {% for region in regions %}
+        <option value="{{ region }}" {{ 'selected' if region == user_region }}>
```

Comment:
This essentially means that the region dropdown in the debug page will only show the 2 letters slugs. The alternative would be to start making XHR calls in the debug page, and I wasn't sure we'd want to go down that route (the less the debug page depends on external resources, the more reliable it is). Thoughts ?